### DISCUSSION TITLE
RFC-0241 Tari Monero atomic swap

### DISCUSSION THREAD

**Discussion by stringhandler** [tari-project/tari#3647] [RFCs]
A discussion to talk about the RFC-0241. It is currently in PR stage in #3623

**Comment by SWvheerden** [comment]
We can remove the DLEQ proof. 
The points are the same, but we need to modify Dalek to confirm they are the same.

**Reply by SWvheerden** [reply]
But if we want to exchange encoded points(or public keys) we need to have some proof.
